### PR TITLE
[1LP][RFR] Adding tests for Ansible Workflow jobs

### PR DESCRIPTION
**Pull request by nachandr**
I have parameterized  the existing tests so that the tests can be run for Ansible Workflow jobs as well.

PRT results 
59 - PASSED
510 - PASSED

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 34ab1a9bb3da2b5596b929a6e00c37a06f96de90:

`cfme/tests/services/test_config_provider_servicecatalogs.py`:
- :bangbang: Line 17:101: **E501** *line too long (111 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/services/test_config_provider_servicecatalogs.py [hunk 1]
```diff
@@ -43,13 +48,16 @@ def config_manager(config_manager_obj):
...
     provider_name = config_manager_obj.yaml_data.get('name')
     provisioning_data = config_manager_obj.yaml_data['provisioning_data']
-    item_type, provider_type, template = map(provisioning_data.get,
+    if job_type == 'template':
+        item_type, provider_type, template = map(provisioning_data.get,
                                             ('item_type', 'provider_type', 'template'))
-
+    else:
+        item_type, provider_type, template = map(provisioning_data.get,
+                                            ('item_type', 'provider_type', 'workflow'))
```
Comment (john-dupuy) [inline cfme/tests/services/test_config_provider_servicecatalogs.py]: Why not just pass `job_type` here? This will reduce LOC.

Comment (jawatts) [reply cfme/tests/services/test_config_provider_servicecatalogs.py]: ^ **Optional**

Comment (mshriver) [reply cfme/tests/services/test_config_provider_servicecatalogs.py]: I'm actually going to call this **Required** - this is unnecessary logic and will just further complicate when we get a 3rd parameter.  Just pass `job_type` to the map call directly.

File: cfme/tests/services/test_config_provider_servicecatalogs.py [hunk 2]
```diff
@@ -63,17 +71,16 @@ def catalog_item(appliance, request, config_manager, dialog, catalog):
     return catalog_item
 
 
-@pytest.mark.tier(2)
-@pytest.mark.ignore_stream('upstream')
-def test_order_tower_catalog_item(appliance, catalog_item, request):
-    """Tests order catalog item
+def test_order_tower_catalog_item(appliance, catalog_item, request, job_type):
```
Comment (mshriver) [inline cfme/tests/services/test_config_provider_servicecatalogs.py]: Why remove the tier marker here? I think both tests should have one.

Comment (jawatts) [reply cfme/tests/services/test_config_provider_servicecatalogs.py]: In the `pytestmark` at the top, all tests are marked with `tier(2)`

File: cfme/tests/services/test_config_provider_servicecatalogs.py (hunk 1 above)
Comment (nachandr) [reply cfme/tests/services/test_config_provider_servicecatalogs.py]: Done

**Comment by dajoRH** [comment]
Lint report for commit 025510485d5dd5b7bedb9d41a7c7ac3a437443af:

`cfme/tests/services/test_config_provider_servicecatalogs.py`:
- :bangbang: Line 55:41: **E231** *missing whitespace after ','*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by nachandr** [comment]
@mshriver, the tiers markers are on line 13.

Could you merge the PR ?

**Pull request merged**